docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -181,6 +181,7 @@ services:
     depends_on:
       - jaeger-all-in-one
 ----
+If `jaeger-all-in-one` fails to start due to `'' has invalid keys: tls` try a fixed version, such as `0.41.0`.
```

Comment:
@manstis I'm having a look at this but I'm not sure I understand. From what I can see this image is at version [1.33](https://hub.docker.com/layers/all-in-one/jaegertracing/all-in-one/1.33/images/sha256-b2b94dda1bd1039925ac7312ba0e1b4ce40ecb097fe936b9bbfa2f3aad2380ad?context=explore) so far beyond 0.41 and we use the `latest`.

What I miss is that if you had issues because you were specifying a version older than 0.41 or if newer versions specified with `latest` have problems?

Thanks!